### PR TITLE
refactor: Simplified import patterns

### PR DESCRIPTION
**Pull request by bennyn**
I turned things like this:

```ts
import AccessTokenData from '../auth/AccessTokenData';
import AuthAPI from '../auth/AuthAPI';
import LoginData from '../auth/LoginData';
```

Into this:
 
```ts
import {AccessTokenData, AuthAPI, LoginData} from '../auth';
```

### CODE REVIEW

**Comment by bennyn** [comment]
The idea was to use `baseUrl` in combination with `paths` but this make the automatically generated `.d.ts` files unusable:

```json
{
  "compilerOptions": {
    "baseUrl": "./",
    "declaration": true,
    "lib": [
      "DOM",
      "ES6"
    ],
    "module": "CommonJS",
    "moduleResolution": "node",
    "noEmitOnError": true,
    "noImplicitAny": true,
    "noImplicitReturns": true,
    "outDir": "dist/commonjs",
    "paths": {
      "*": [
        "./src/ts/*"
      ]
    },
    "removeComments": true,
    "rootDir": "src/ts",
    "sourceMap": false,
    "target": "ES5"
  },
  "exclude": [
    "dist/commonjs",
    "node_modules"
  ]
}
```

So I just created some `index.ts` files to make importing multiple classes easier.

File: src/ts/auth/AuthAPI.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import {AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios';
 
-import AccessTokenData from './AccessTokenData';
-import HttpClient from '../http/HttpClient';
-import LoginData from './LoginData';
+import * as auth from '../auth';
```
Comment (Yserz) [inline src/ts/auth/AuthAPI.ts]: Maybe `import {AccessTokenData, Login} from '.../auth';`

File: src/ts/team/TeamAPI.ts [hunk 2]
```diff
@@ -1,10 +1,7 @@
 import {AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios';
 
-import HttpClient from '../http/HttpClient';
-import MemberData from './MemberData';
-import NewTeamData from './NewTeamData';
-import TeamChunkData from './TeamChunkData';
-import TeamData from './TeamData';
+import * as team from '../team';
```
Comment (Yserz) [inline src/ts/team/TeamAPI.ts]: Maybe `import {NewTeamData, TeamData, TeamChunkData, MemberData} from '.../team';`

Comment (bennyn) [inline src/ts/team/TeamAPI.ts]: But that's why I used `*`. Because `NewTeamData, TeamData, TeamChunkData, MemberData` are all classes available.

Comment (herrmannplatz) [inline src/ts/team/TeamAPI.ts]: +1 for the explicit imports

Makes it also easier for webpack to tree shake. Not the most important thing in this case but still.

Comment (bennyn) [inline src/ts/team/TeamAPI.ts]: https://vote4code.com/5685057352105984/explicit-imports 😆

**Pull request merged**